Marlin/Configuration.h

```diff
@@ -1535,7 +1535,7 @@
  *     O-- FRONT --+
  */
 #if ENABLED(KNUTWURST_BLTOUCH)
-  #define NOZZLE_TO_PROBE_OFFSET { -2, -25, -0.4 } //https://www.thingiverse.com/thing:2824005
+  #define NOZZLE_TO_PROBE_OFFSET { 0, -43, -1.35 } //https://www.thingiverse.com/thing:2824005
```

Comment:
See above. This can only be a default for the 4MAX, not all other printers.

Reply:
This is right. But would be very near for both machines if the BLTouch in installed inside the printhead to not reduce the printbedsize.